src/ol/source/imagestaticsource.js

```diff
@@ -42,16 +37,34 @@ ol.source.ImageStatic = function(options) {
   goog.base(this, {
     attributions: attributions,
     logo: options.logo,
-    projection: ol.proj.get(options.projection),
-    resolutions: resolutions
+    projection: ol.proj.get(options.projection)
   });
 
   /**
    * @private
    * @type {ol.Image}
    */
-  this.image_ = new ol.Image(imageExtent, resolution, 1, attributions,
+  this.image_ = new ol.Image(imageExtent, undefined, 1, attributions,
       options.url, crossOrigin, imageLoadFunction);
+
+  goog.events.listen(this.image_, goog.events.EventType.CHANGE, function() {
```

Comment:
There are now two listeners for `CHANGE`.  Is it important that this listener get called first?

What would you think about registering a single listener (`this.handleImageChange_`) and having that call the parent method (`this.handleImageChange`)?